Sources/Networking/HTTPClient.swift

```diff
@@ -142,7 +146,8 @@ private extension HTTPClient {
             "X-Client-Version": SystemInfo.appVersion,
             "X-Client-Build-Version": SystemInfo.buildVersion,
             "X-StoreKit2-Setting": "\(self.systemInfo.storeKit2Setting.debugDescription)",
-            "X-Observer-Mode-Enabled": observerMode
+            "X-Observer-Mode-Enabled": "\(observerMode)",
+            "X-Is-Sandbox": "\(sandbox)"
```

Comment:
this could now be simplified to
```suggestion
            "X-Is-Sandbox": "\(self.systemInfo.isSandbox)"
```

Reply:
Changed.